### PR TITLE
[ZT] update CORS config instructions

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

**Comment by jroyal** [comment]
LGTM!

File: content/cloudflare-one/policies/access/cors.md [hunk 1]
```diff
@@ -1,102 +1,245 @@
...
+- Do not troubleshoot CORS in Incognito mode, as this will cause disruptions with Access due to `CF-Authorization` being blocked as a third-party cookie on cross origin requests.
 
-- Do not troubleshoot CORS in Incognito mode, as this will cause disruptions with Access due to `CF-Authorization` being blocked as a third-party cookie on cross origin requests"
+- Safari, in particular Safari 13.1, handles cookies in a unique format. In some cases, this can cause CORS to fail. This will be dependent on Apple releasing a patch for handling cookies. This is known to impact macOS 10.15.4 when running Safari 13.1 (15609.1.20.111.8).
 
 {{</Aside>}}
 
-If you have two sites protected by Cloudflare Access, `app1.site.com` and `app2.hostname.com`, requests made between the two will be subject to CORS checks. Users who login to `app1.site.com` will be issued cookies for `app1.site.com` and your Cloudflare Access [team domain](/cloudflare-one/glossary/#team-domain).
+## Allow Simple Requests
```
Comment (abracchi-tw) [inline content/cloudflare-one/policies/access/cors.md]:
```suggestion
## Allow simple requests
```

File: content/cloudflare-one/policies/access/cors.md [hunk 2]
```diff
@@ -1,102 +1,245 @@
...
+
+If you make a simple cross-origin request to an Access-protected domain and have not yet logged in, the request will return a `CORS error`. There are two ways you can resolve this error:
```
Comment (abracchi-tw) [inline content/cloudflare-one/policies/access/cors.md]: should we stick to either `CORS request` or `cross-origin request`?

File: content/cloudflare-one/policies/access/cors.md [hunk 3]
```diff
@@ -1,102 +1,245 @@
...
+## Allow Preflighted Requests
+
+If you make a preflighted cross-origin request to an Access-protected domain, the OPTIONS request will return a `403` error. This error occurs regardless of whether you have logged in to the domain. This is because the browser never includes cookies with OPTIONS requests, by design. Cloudflare will therefore block the preflight request, causing the CORS exchange to fail.
+
+There are two ways you can resolve this error:
+
+- **Option 1** — [Configure Cloudflare to respond to the OPTIONS request](#configure-response-to-preflight-request).
+- **Option 2** — [Create a Cloudflare Worker which automatically sends an authentication token](#send-authentication-token-with-cloudflare-worker). This method only works if both sites involved in the CORS exchange are behind Access.
+
+### Configure response to preflight request
```
Comment (abracchi-tw) [inline content/cloudflare-one/policies/access/cors.md]: should `responses` be plural? do we have different types? also, should it be `preflight requests`?

File: content/cloudflare-one/policies/access/cors.md [hunk 4]
```diff
@@ -1,102 +1,245 @@
...
+3. Click the **Policies** tab.
+
+4. Add the following policy:
+| Action        | Rule type | Selector      |
+| ------------- | --------- | ------------- |
+| Service Auth  | Include   | Service Token |
+
+### 3. Create a new Worker
+
+1. Open a terminal and create a new worker project.
```
Comment (abracchi-tw) [inline content/cloudflare-one/policies/access/cors.md]:
```suggestion
1. Open a terminal and create a new Workers project.
```
not sure though

**Review by abracchi-tw — APPROVED** [review]

File: content/cloudflare-one/policies/access/cors.md (hunk 3 above)
Comment (ranbel) [reply content/cloudflare-one/policies/access/cors.md]: I think response should be singular (you can only configure one per domain), but request should be plural (you can send multiple requests to a domain).

File: content/cloudflare-one/policies/access/cors.md [hunk 5]
```diff
@@ -1,102 +1,245 @@
...
+## Allow Preflighted Requests
+
+If you make a preflighted cross-origin request to an Access-protected domain, the OPTIONS request will return a `403` error. This error occurs regardless of whether you have logged in to the domain. This is because the browser never includes cookies with OPTIONS requests, by design. Cloudflare will therefore block the preflight request, causing the CORS exchange to fail.
+
+There are two ways you can resolve this error:
+
+- **Option 1** — [Configure Cloudflare to respond to the OPTIONS request](#configure-response-to-preflight-request).
+- **Option 2** — [Create a Cloudflare Worker which automatically sends an authentication token](#send-authentication-token-with-cloudflare-worker). This method only works if both sites involved in the CORS exchange are behind Access.
+
+### Configure response to preflight request
```
Comment (ranbel) [inline content/cloudflare-one/policies/access/cors.md]:
```suggestion
### Configure response to preflight requests
```

File: content/cloudflare-one/policies/access/cors.md [hunk 6]
```diff
@@ -1,102 +1,245 @@
...
 
-- Do not troubleshoot CORS in Incognito mode, as this will cause disruptions with Access due to `CF-Authorization` being blocked as a third-party cookie on cross origin requests"
+- Safari, in particular Safari 13.1, handles cookies in a unique format. In some cases, this can cause CORS to fail. This will be dependent on Apple releasing a patch for handling cookies. This is known to impact macOS 10.15.4 when running Safari 13.1 (15609.1.20.111.8).
 
 {{</Aside>}}
 
-If you have two sites protected by Cloudflare Access, `app1.site.com` and `app2.hostname.com`, requests made between the two will be subject to CORS checks. Users who login to `app1.site.com` will be issued cookies for `app1.site.com` and your Cloudflare Access [team domain](/cloudflare-one/glossary/#team-domain).
+## Allow simple requests
+
+If you make a simple cross-origin request to an Access-protected domain and have not yet logged in, the request will return a `CORS error`. There are two ways you can resolve this error:
```
Comment (ranbel) [inline content/cloudflare-one/policies/access/cors.md]:
```suggestion
If you make a simple CORS request to an Access-protected domain and have not yet logged in, the request will return a `CORS error`. There are two ways you can resolve this error:
```

**Pull request merged**